script/app.js

```diff
@@ -33,11 +33,17 @@ const getTodoCard = (task) => {
     element.innerHTML = `
         <h1>${task.title}</h1>
        <p>Created At: ${formatDate()}</p>
-        <button class="task-card__icon">Complete</button>
-        <button class="task-card__icon">Edit</button>
-        <button class="task-card__icon">Delete</button>
+        <button class="task-card__icon" id="complete">Complete</button>
```

Comment:
let's not both id and class. I would suggest to use multiple classname except using id